dietpi/dietpi-software

```diff
@@ -3612,41 +3613,16 @@ We work around this error by running APT a second time. Please do not worry and
 			#Stretch
 			elif (( $G_DISTRO == 4 )); then
 
-				INSTALL_URL_ADDRESS='https://dietpi.com/downloads/binaries/all/mpd_0.20.23-1_'
-				#armv6
-				if (( $G_HW_ARCH == 1 )); then
-
-					INSTALL_URL_ADDRESS+='armv6.deb'
-
-				#armv7+
-				elif (( $G_HW_ARCH == 2 )); then
-
-					INSTALL_URL_ADDRESS+='armv7.deb'
-
-				#ARMv8
-				elif (( $G_HW_ARCH == 3 )); then
-
-					INSTALL_URL_ADDRESS+='armv8.deb'
-
-				#x86_64
-				elif (( $G_HW_ARCH == 10 )); then
-
-					INSTALL_URL_ADDRESS+='amd64.deb'
-
-				fi
-
-				apt-mark unhold mpd &> /dev/null #??? Not required for dpkg -i installs
-
 				DEPS_LIST='libnfs8 libsmbclient libsqlite3-0 libupnp6 libwrap0 libmpdclient2 libflac8 libyajl2 libavahi-client3 libvorbisfile3 libwavpack1 libmad0 libmpg123-0 libopus0 libavformat57 libfaad2 libcdio-paranoia1 libiso9660-8 libshout3 libid3tag0'
-				Download_Install "$INSTALL_URL_ADDRESS"
-
-				apt-mark hold mpd # prevent repo updates from overwriting
+				Download_Install "https://dietpi.com/downloads/binaries/all/mpd_0.20.23-1_$G_HW_ARCH_DESCRIPTION.deb"
```

Comment:
@MichaIng 

I completely forgot about ```$G_HW_ARCH_DESCRIPTION.deb```, nice one 👍

Reply:
Jep, reduces line count very much. We should gradually apply this to all binaries, if we anyway update them.